packages/survey-core/src/survey.ts

```diff
@@ -6391,13 +6397,20 @@ export class SurveyModel extends SurveyElementCore
     if (!json) return;
     this.questionHashesClear();
     this.jsonErrors = null;
+    this.sjsVersion = undefined;
     const jsonConverter = new JsonObject();
     jsonConverter.toObject(json, this, options);
     if (jsonConverter.errors.length > 0) {
       this.jsonErrors = jsonConverter.errors;
     }
     this.onStateAndCurrentPageChanged();
     this.updateState();
+    if(!!this.sjsVersion && !!settings.version) {
+      if(Helpers.compareVerions(this.sjsVersion, settings.version) > 0) {
+        ConsoleWarnings.warn("The JSON was created using Creator version "
```

Comment:
"The version of the survey JSON schema (v1.11.11) is newer than your current Form Library version (v1.11.10). Please update the Form Library to make sure that all survey features work as expected."